NOTICE

```diff
@@ -10,14 +10,44 @@ may not be used or distributed or accessed in any manner other than directly
 via Code.org's own web site, mobile applications, or social media channels,
 as-is without  modification or embedding in another format.
```

Comment:
Updated to include the same list of materials, but otherwise sticking to the same language as before.